scripts/mybgg/bgg_client.py

```diff
@@ -245,7 +357,8 @@ def log_item(_, item):
                                     ], required=False),
                                     hooks=xml.Hooks(after_parse=numplayers_to_result)
                                 )
-                            ]),
+                            ],
+                            required=False),
```

Comment:
I guess this fixes some bug with some games not having suggested_numplayers? Would love this aswell.